### PR TITLE
Add rect for window

### PR DESCRIPTION
**Pull request by KazuCocoa**
This implementation is for Android driver.
As same as : https://github.com/appium/appium-xcuitest-driver/pull/607

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -76,6 +76,15 @@ commands.getWindowSize = function () {
   return this.bootstrap.sendAction('getDeviceSize');
 };
 
+// For W3C
+commands.getWindowRect = function () {
```
Comment (mykola-mokhnach) [inline lib/commands/general.js]: is the endpoint present in NO_PROXY list?

Comment (imurchie) [reply lib/commands/general.js]: This probably does need to be added, or else it won't work in Chrome (unless Chromedriver handles this endpoint?).

Comment (mykola-mokhnach) [reply lib/commands/general.js]: then there should be isWebContext verification?

Comment (imurchie) [reply lib/commands/general.js]: Window size should be the native window size, no?

Comment (KazuCocoa) [reply lib/commands/general.js]: Yes, it should be window size on mobile devices so far.

File: test/unit/commands/general-specs.js [hunk 2]
```diff
@@ -120,6 +120,13 @@ describe('General', function () {
       await driver.getWindowSize().should.be.equal('size');
     });
   });
+  describe('getWindowRect', function () {
+    it('should get window size', async () => {
+      sandbox.stub(driver.bootstrap, 'sendAction')
+        .withArgs('getDeviceSize').returns({width: 300, height: 400});
+      await driver.getWindowRect().toString().should.be.equal({width: 300, height: 400, x: 0, y: 0}.toString());
```
Comment (mykola-mokhnach) [inline test/unit/commands/general-specs.js]: toString() will return something like `[object Object]`. You need to wrap objects into JSON.stringify to reach proper results

**Pull request merged**